deploy/crds/crd-certificaterequests.yaml

```diff
@@ -54,7 +54,7 @@ spec:
           type: date
       schema:
         openAPIV3Schema:
-          description: "A CertificateRequest is used to request a signed certificate from one of the configured issuers. \n All fields within the CertificateRequest's `spec` are immutable after creation. A CertificateRequest will either succeed or fail, as denoted by its `status.state` field. \n A CertificateRequest is a 'one-shot' resource, meaning it represents a single point in time request for a certificate and cannot be re-used."
+          description: "A CertificateRequest is used to request a signed certificate from one of the configured issuers. \n All fields within the CertificateRequest's `spec` are immutable after creation. A CertificateRequest will either succeed or fail, as denoted by its `status.state` field. \n A CertificateRequest is a `one-shot` resource, meaning it represents a single point in time request for a certificate and cannot be re-used."
```

Comment:
i would drop the quotes on one-shot as it is not a field name

Reply:
Dropped the quote for all "one-shot" - thanks!